_posts/example-2.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Example 2 post'
+excerpt: 'Example 2 post Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Praesent elementum facilisis leo vel fringilla est ullamcorper eget. At imperdiet dui accumsan sit amet nulla facilities morbi tempus.'
+coverImage: '/assets/posts/example/cover.png'
+date: '2022-05-15T15:00:00+07:00'
+author:
+  name: Hoang
+  picture: '/assets/authors/hoang.jpeg'
```

Comment:
I cannot comment on an image 🤣
Just want to recommend you to use https://tinypng.com/ or https://www.npmjs.com/package/tinypng-cli for image optimizing 🖖

Reply:
Cool, I optimized images in https://github.com/nimblehq/til-web/pull/8/commits/f27a03e6300dccac6890170088a33f02891e45c2